apollo-runtime/src/main/java/com/apollographql/apollo/ApolloClient.java

```diff
@@ -195,14 +192,10 @@ public Builder httpCache(@Nonnull ResponseCacheStore cacheStore, @Nonnull Evicti
     /**
      * Set the configuration to be used for normalized cache.
      *
-     * @param cacheStore       the cache store to use
-     * @param cacheKeyResolver the cache key resolver
      * @return The {@link Builder} object to be used for chaining method calls
      */
-    public Builder normalizedCache(@Nonnull CacheStore cacheStore,
-        @Nonnull CacheKeyResolver<Map<String, Object>> cacheKeyResolver) {
-      this.cache = new RealCache(checkNotNull(cacheStore, "cacheStore == null"),
-          checkNotNull(cacheKeyResolver, "cacheKeyResolver == null"));
+    public Builder normalizedCache(@Nonnull Cache cache) {
```

Comment:
I guess we need to revert this back, as RealCache is supposed to be hidden from user (it's under internal package). Unfortunately java doesn't allow to make it internal per jar file. But the idea that user should not deal with any class from apollo internal package